### PR TITLE
🎁 Working PDF Split

### PR DESCRIPTION
**Pull request by orangewolf**
Brings the split_ocr_thumbnail to fully functioning. Using the sample script, we can go from a ADL url to individual tiff files across two steps and queue up thumbnail and ocr jobs for those steps once they are written.

### CODE REVIEW

File: awslambda/Dockerfile [hunk 1]
```diff
@@ -152,7 +170,8 @@ RUN cp -vR */usr/* /opt
 # The x86_64 packages extract as lib64, we need to move these files to lib
 RUN yum install -y rsync
 RUN rsync -av /opt/lib64/ /opt/lib/
-RUN rm -r /opt/lib64
+RUN rm -r /opt/lib64 && \
+  unlink /opt/lib/libGLX_system.so.0 && cp /usr/lib64/libGLX_mesa.so.0 /opt/lib/libGLX_system.so.0
```
Comment (orangewolf) [inline awslambda/Dockerfile:174]: one of the libs is a symlink. we have to break the symlink and make it a copy or pdfinfo just crashes.

File: awslambda/bin/sample_post.rb [hunk 2]
```diff
@@ -0,0 +1,83 @@
+require 'httparty'
```
Comment (orangewolf) [inline awslambda/bin/sample_post.rb:1]: this can likely be deleted down the road, but is pretty useful at the moment.

File: awslambda/Dockerfile [hunk 3]
```diff
@@ -152,7 +170,8 @@ RUN cp -vR */usr/* /opt
 # The x86_64 packages extract as lib64, we need to move these files to lib
 RUN yum install -y rsync
 RUN rsync -av /opt/lib64/ /opt/lib/
-RUN rm -r /opt/lib64
+RUN rm -r /opt/lib64 && \
```
Comment (jeremyf) [inline awslambda/Dockerfile:173]:
```suggestion
# One of the libs is a symlink. we have to break the symlink and make it a copy or pdfinfo just crashes.
RUN rm -r /opt/lib64 && \
```

File: awslambda/handler.rb [hunk 4]
```diff
@@ -126,14 +136,39 @@ def download_to_tmp(input_uri:, output_location_template: 'file:///tmp/{{dir_par
...
+# @api private
+#
+# Convert SQS url from Amazons format to Derivative Rodeo's format. Add template information and s3 destination
+#
+# @param queue_url [String]
+# @return [String]
+def queue_url_to_rodeo_url(queue_url:, s3_url: nil, template: nil)
+  url = queue_url.gsub('https://sqs.', 'sqs://')
+
+  url += "/#{template}" if template
```
Comment (jeremyf) [inline awslambda/handler.rb:161]: I'm a bit confused in reading lines 161 and 162; if we receive both `:template` and `:s3_url` we'll have a URL of the form:  `sqs://HOST/:template/?template=:s3_url/:template` that is the given `:template` will be twice in the returned URL.  Is that the desired behavior?

**Review by jeremyf — APPROVED** [review]

**Pull request merged**